basics/demo/components/layout.js

```diff
@@ -63,8 +61,8 @@ export default function Layout({ children, home }) {
       <main>{children}</main>
       {!home && (
         <div className={styles.backToHome}>
-          <Link href="/">
-            <a>← Back to home</a>
+          <Link legacyBehavior href="/">
```

Comment:
Since the anchor tag was removed, this probably shouldn't have the `legacyBehaviour` flag.

Reply:
Updated layout.js to correct this. 
Good catch!